test/__snapshots__/index.test.js.snap

```diff
@@ -310,12 +368,12 @@ esnext-ts
 
 
 Build \\"esnextTs\\" to dist:
-470 B: esnext-ts.js.gz
-404 B: esnext-ts.js.br
-470 B: esnext-ts.mjs.gz
-409 B: esnext-ts.mjs.br
-528 B: esnext-ts.umd.js.gz
-461 B: esnext-ts.umd.js.br"
+841 B: esnext-ts.js.gz
```

Comment:
While the size increase is looking substantial, we were previously not transpiling all `asnyc/await` keywords in the example. This PR fixes that